packages/framework/src/Form/Admin/Customer/DeliveryAddressFormType.php

```diff
@@ -1,188 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shopsys\FrameworkBundle\Form\Admin\Customer;
-
-use Shopsys\FrameworkBundle\Form\GroupType;
-use Shopsys\FrameworkBundle\Form\ValidationGroup;
-use Shopsys\FrameworkBundle\Model\Country\CountryFacade;
-use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressData;
-use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
-use Symfony\Component\Form\Extension\Core\Type\FormType;
-use Symfony\Component\Form\Extension\Core\Type\TextType;
-use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\Form\FormInterface;
-use Symfony\Component\OptionsResolver\OptionsResolver;
-use Symfony\Component\Validator\Constraints;
-
-class DeliveryAddressFormType extends AbstractType
```

Comment:
`project-base/app/src/Form/Admin/Customer/DeliveryAddressFormTypeExtension.php` should be removed too, along with the reference in `project-base/app/config/forms.yaml`

Reply:
true, fixed https://github.com/shopsys/shopsys/pull/2494/commits/73b18de19f53e2cac538b2ab7e68f5e01dc7b2bc